### PR TITLE
Add driver and opt flags for network create

### PR DESCRIPTION
**Pull request by Junnplus**
Signed-off-by: ye.sijun <junnplus@gmail.com>

ref: https://github.com/containerd/nerdctl/issues/692

- [x] need test

### CODE REVIEW

**Comment by Junnplus** [comment]
@AkihiroSuda Please take a look :)

File: README.md [hunk 1]
```diff
@@ -908,12 +908,14 @@ Create a network
 Usage: `nerdctl network create [OPTIONS] NETWORK`
 
 Flags:
+- :whale: `-d, --driver`: Driver to manage the Network
+- :whale: `-o, --opt`: Set driver specific options
```
Comment (AkihiroSuda) [inline README.md:912]: Could you explain the list of the drivers and the opts in this documentation?

Comment (Junnplus) [reply README.md:914]: Done.

**Comment by AkihiroSuda** [comment]
Windows CI failing https://cirrus-ci.com/task/5470749778509824

> level=fatal msg="invalid argument"

File: README.md [hunk 2]
```diff
@@ -908,12 +908,18 @@ Create a network
 Usage: `nerdctl network create [OPTIONS] NETWORK`
 
 Flags:
+- :whale: `-d, --driver=(bridge|nat)`: Driver to manage the Network
+    - :whale: `--driver=bridge`: Default driver for unix
+    - :nerd_face: :blue_square: `--driver=nat`: Default driver for windows
+- :whale: `-o, --opt`: Set driver specific options
+    - :whale: `--opt=com.docker.network.driver.mtu=<MTU>`: Set the containers network MTU
+    - :nerd_face: `--opt=mtu=<MTU>`: Alias of `--opt=com.docker.network.driver.mtu=<MTU>`
```
Comment (AkihiroSuda) [inline README.md:916]: s/🤓/🐳/

**Comment by Junnplus** [comment]
> Windows CI failing

green.

File: pkg/netutil/netutil_unix.go [hunk 3]
```diff
@@ -27,18 +27,30 @@ const (
...
+		mtu := 0
+		for opt, v := range opts {
+			switch opt {
+			case "mtu", "com.docker.network.driver.mtu":
+				mtu, err = ParseMTU(v)
+				if err != nil {
+					return nil, err
+				}
+			default:
+				return nil, fmt.Errorf("unsupported network option %q", opt)
```
Comment (fahedouch) [inline pkg/netutil/netutil_unix.go:49]: do you plan to support the rest of the network options  (https://docs.docker.com/engine/reference/commandline/network_create/#bridge-driver-options) in a future PR  ?

Comment (Junnplus) [reply pkg/netutil/netutil_unix.go:49]: Of course.

Comment (Junnplus) [reply pkg/netutil/netutil_unix.go:49]: Of course, it may take a while

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**